<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>559</width>
    <height>411</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menedżer Obiektów</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>301</width>
      <height>225</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="localNameKey">
       <property name="text">
        <string>Nazwa lokalizacji:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="localNameVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="localAdressKey">
       <property name="text">
        <string>Adres lokalizacji:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="localAdressVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="roomKey">
       <property name="text">
        <string>Pomieszczenie:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="roomVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="battTypeKey">
       <property name="text">
        <string>Typ baterii:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="battTypeVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="cellNumberKey">
       <property name="text">
        <string>Ilość ogniw:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="cellNumberVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="messEndVoltKey">
       <property name="text">
        <string>Napięcie końca pomiaru:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="messEndVoltVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="installDataKey">
       <property name="text">
        <string>Data instalacji baterii:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="installDataVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="messMadeCurrKey">
       <property name="text">
        <string>Pomiar wykonano prądem:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="messMadeCurrVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="roomTempKey">
       <property name="text">
        <string>Temperatura pomieszczenia:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="roomTempVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="extCellStateKey">
       <property name="text">
        <string>Zewnętrzny stan ogniw:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="extCellStateVal">
       <property name="text">
        <string>def</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="commentsKey">
       <property name="text">
        <string>Uwagi:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="commentsVal">
       <property name="text">
        <string>def</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Wykonane pomiary:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
